src/libs/client.js

```diff
@@ -468,3 +468,48 @@ export const removeNotificationDeviceToken = async client => {
 
   await saveClient(client)
 }
+
+export const getOidcAccessToken = async (client, oidcCode) => {
```

Comment:
this method does not do what it says it does. It doesn't return the `access_token`. Also this method is async and make a request to the stack, should be `fetchSomething`.

Reply:
What about `connectOidcClient()` since the method is inspired from `connectClient()` one?

Reply:
👌

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/c5471ef90602aabec1cb57301e0abcadedcfdadd..2542fadaa6f412b928aa438258e603e51a74aabd